Sources/PixelEditor/Components/StepSlider.swift

```diff
@@ -23,6 +23,12 @@ import Foundation
 import PixelEngine
 
 
+extension CGSize {
+    static func *(lhs: CGSize, rhs: CGFloat) -> CGSize {
```

Comment:
@adamnemecek 

Sorry, I did mistake commenting.

> muukii commented on Sources/PixelEditor/Components/ImageScrollView.swift in 7ee125f 8 hours ago
I think this extension is too much.
I don’t like adding extensions.
It is necessary to carefully judge whether it is really necessary or not.
Extensions will be shorter some code.but side-effect will be bigger than some merits.

> adamnemecek replied 7 hours ago
I find extensions to be the cleanest way to add functionality. in this case, it's a very straight forward extension, it's something that should really come with CGRect by default.

---

I know what you think. But it depends on each project.
Definitely, the code will be cleaned up by that extension.
But, other developers need to know custom extensions when they contribute.
In this project, I think no-extension makes some developers can look only the code where they want to change.
On the other hands, if there are so many verbose codes, we need to necessary extensions.
That is a tradeoff.
Summary, in this case and project, I would not like to add some extension to built-in libraries.